coordinator/ethereum/test-utils/src/main/kotlin/net/consensys/zkevm/ethereum/MakefileContractDeploymentHelper.kt

```diff
@@ -75,10 +75,10 @@ fun executeCommand(
 }
 
 internal val lineaRollupAddressPattern = Pattern.compile(
-  "^LineaRollup(?:.*)? deployed: address=(0x[0-9a-fA-F]{40}) blockNumber=(\\d+)"
+  "^contract=LineaRollup(?:.*)? deployed: address=(0x[0-9a-fA-F]{40}) blockNumber=(\\d+)"
 )
 internal val l2MessageServiceAddressPattern = Pattern.compile(
-  "^L2MessageService(?:.*)? deployed: address=(0x[0-9a-fA-F]{40}) blockNumber=(\\d+)"
+  "contract=L2MessageService(?:.*)? deployed: address=(0x[0-9a-fA-F]{40}) blockNumber=(\\d+)"
```

Comment:
inconsistency, one forces Line start whereas the other not. Maybe we can remove the "^"